cpu/esp_common/Makefile.dep

```diff
@@ -93,3 +93,7 @@ ifneq (,$(filter esp_wifi_any,$(USEMODULE)))
   USEMODULE += netopt
   USEMODULE += ztimer_msec
 endif
+
+ifneq (,$(filter ztimer_msec,$(USEMODULE)))
+  USEMODULE += core_thread_flags
```

Comment:
Isn't that much more a dependency of the FreeRTOS compat module?

Reply:
I'm not sure. The FreeRTOS compat module requires `core_thread_flags` only if `ztimer_msec` is used to realize the timeouts in `xQueue`. This in turn is only required, when network interfaces are used (if I remember correctly) and `ztimer_msec` is enabled in that case anyway. On the other hand, `ztimer_msec` doesn't require `core_thread_flags` usually.

The correct condition would be if `esp_freertos` && `ztimer_msec` && network interfaces are used, then we can realize timeouts and require `core_thread_flags` for that. Since `esp_freertos` is always enabled, the condition as is is the result.